### ISSUE TITLE
Disconnecting my character

### ISSUE THREAD

**Issue by Theddasapa** [WeakAuras/WeakAuras2#3207]
### Description

I am not sure what exactly about WeakAuras is disconnecting my main (only my main) character the last 2-3 days, but I have narrowed it down to this addon.  I uninstalled/reinstalled wow, deleted all the folders including cache/interface/wft folders, uninstalled/reinstalled my addon manager (Wowup.io) and I was able to log into my character without disconnecting. I started adding addon again one by one and testing them and it was when I added WeakAuras that it started happening again.

### WeakAuras Version

WeakAuras 9.1.0

### World of Warcraft Flavor

Retail (Default)

### Tested with only WeakAuras

- [X] Yes
- [ ] No

### Reproduction Steps

1. Log into main character
2. Immediately disconnected.

### Last Good Version

WeakAuras 9.1.0

### Export String

All of my weak aura was erased, it is the addon itself causing the disconnection issue.

**Comment by mrbuds** [comment]
We can't help with so few information, as asked in the issue template, we need Lua errors, your savedvariable file, and the class/spec of your main to have a better idea of which of your auras are loaded.

**Comment by Theddasapa** [comment]
Yeah, I cannot really help you there... I am just trying to let you know
that something is up.  I deleted the addon and all information with it to
prevent my disconnecting issues so I do not have any of that info. My
character is a boomkin and it was disconnecting even when I had no auras at
all loaded.  Someone suggested trying a second WeakAurass, and I did not
notice that there was one with a purple icon and one with a black icon, and
the black icon is the one that is most downloaded (and I was using the
purple icon one), and it has not given me a problem so far although I still
have not loaded any new auras to it.

Doug
